packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -486,6 +489,113 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
+  // Show the month picker over date picker (1 column layout), if enabled
```

Comment:
Oops, I didn't realize that there were all these .scss files, they shouldn't be here.  I will get rid of them.  They aren't apart of this PR.